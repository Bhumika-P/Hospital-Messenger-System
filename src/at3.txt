list_groups
list_users
add_user(1,"")
add_user(0, "aaa")
add_user(1," ")
add_user(0," ")
add_user(0,"")
add_group(0, "aaa")
add_group(1, "")
add_group(1, " ")
add_group(0, "")
register_user(0,0)
register_user(1,0)
register_user(5,10)
register_user(0,5)
register_user(0,1)
register_user(1,5)
list_new_messages(0)
list_new_messages(1)
list_new_messages(15)
list_old_messages(0)
list_old_messages(1)
list_old_messages(15)
read_message(0,1)
read_message(1,0)
read_message(1,5)
read_message(0,5)
read_message(5,5)
delete_message(0,0)
delete_message(5,0)
delete_message(0,5)
delete_message(5,5)