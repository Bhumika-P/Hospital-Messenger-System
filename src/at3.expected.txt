  0:  OK

->list_groups
  1:  OK
  There are no groups registered in the system yet.

->list_users
  2:  OK
  There are no users registered in the system yet.

->add_user(1,"")
  3:  ERROR 
  User name must start with a letter.

->add_user(0,"aaa")
  4:  ERROR 
  ID must be a positive integer.

->add_user(1," ")
  5:  ERROR 
  User name must start with a letter.

->add_user(0," ")
  6:  ERROR 
  ID must be a positive integer.

->add_user(0,"")
  7:  ERROR 
  ID must be a positive integer.

->add_group(0,"aaa")
  8:  ERROR 
  ID must be a positive integer.

->add_group(1,"")
  9:  ERROR 
  Group name must start with a letter.

->add_group(1," ")
  10:  ERROR 
  Group name must start with a letter.

->add_group(0,"")
  11:  ERROR 
  ID must be a positive integer.

->register_user(0,0)
  12:  ERROR 
  ID must be a positive integer.

->register_user(1,0)
  13:  ERROR 
  ID must be a positive integer.

->register_user(5,10)
  14:  ERROR 
  User with this ID does not exist.

->register_user(0,5)
  15:  ERROR 
  ID must be a positive integer.

->register_user(0,1)
  16:  ERROR 
  ID must be a positive integer.

->register_user(1,5)
  17:  ERROR 
  User with this ID does not exist.

->list_new_messages(0)
  18:  ERROR 
  ID must be a positive integer.

->list_new_messages(1)
  19:  ERROR 
  User with this ID does not exist.

->list_new_messages(15)
  20:  ERROR 
  User with this ID does not exist.

->list_old_messages(0)
  21:  ERROR 
  ID must be a positive integer.

->list_old_messages(1)
  22:  ERROR 
  User with this ID does not exist.

->list_old_messages(15)
  23:  ERROR 
  User with this ID does not exist.

->read_message(0,1)
  24:  ERROR 
  ID must be a positive integer.

->read_message(1,0)
  25:  ERROR 
  ID must be a positive integer.

->read_message(1,5)
  26:  ERROR 
  User with this ID does not exist.

->read_message(0,5)
  27:  ERROR 
  ID must be a positive integer.

->read_message(5,5)
  28:  ERROR 
  User with this ID does not exist.

->delete_message(0,0)
  29:  ERROR 
  ID must be a positive integer.

->delete_message(5,0)
  30:  ERROR 
  ID must be a positive integer.

->delete_message(0,5)
  31:  ERROR 
  ID must be a positive integer.

->delete_message(5,5)
  32:  ERROR 
  User with this ID does not exist.
